File: Fizzbuzz/python/try.02/fizzbuzz.py
def fizzbuzz(num):
    return [fizzbuzz_gen(x) for x in range(1, num + 1)]

def fizzbuzz_gen(num):
    ret_val = ''

    if num % 3 == 0:
        ret_val += 'fizz'
    
    if num % 5 == 0:
        ret_val += 'buzz'

    if ret_val == '':
        ret_val = num

    return ret_val

File: Fizzbuzz/Android/Fizzbuzz2/app/src/main/java/com/tonytroy/fizzbuzz/Fizzbuzz.java
package com.tonytroy.fizzbuzz;

import java.util.TreeMap;

/**
 * Created by TT on 4/14/2016.
 */
public class Fizzbuzz {
    public TreeMap<Integer, String> magicNumbersMap;

    public Fizzbuzz() {
        magicNumbersMap = new TreeMap<>();
        magicNumbersMap.put(3, "fizz");
        magicNumbersMap.put(5, "buzz");
        magicNumbersMap.put(15, "fizzbuzz");
    }

    public String[] Create(int listSize) {
        String[] FizzBuzzList = new String[listSize + 1];
        for(int i=0; i <= listSize; i++){
            FizzBuzzList[i] = GetFizzBuzzValue(i);
        }

        return FizzBuzzList;
    }

    private String GetFizzBuzzValue(int intValue) {
        String fizzBuzzValue = String.format("%d", intValue);

        for (int magicNumber : magicNumbersMap.descendingKeySet()) {
            if (intValue % magicNumber == 0) {
                fizzBuzzValue = magicNumbersMap.get(magicNumber);
                break;
            }
        }

        return fizzBuzzValue;
    }
}
